pages/user guides/discover_nmap.md

```diff
@@ -8,58 +8,53 @@ permalink: discover-nmap.html
 
 
 ## Using Nmap with Tidal Tools
- 
-By leveraging the power of **Tidal Tools** you can send the output generated by Nmap to your Tidal Migrations account.
- 
+By leveraging the power of **Tidal Tools**, you can send the output generated by Nmap to your Tidal Migrations account.
 1. Install [Tidal Tools](https://guides.tidalmg.com/tidal-tools.html)
-2. [Login to Tidal Tools](https://guides.tidalmg.com/tidal-tools.html#login) with **tidal login**
-3. Run Nmap ([Need to install Nmap?](#installing-nmap)) with the flags of your choosing and save the output to an XML file. For example, 
+2. [Login to Tidal Tools](https://guides.tidalmg.com/tidal-tools.html#login) with **Tidal login**
+3. Run Nmap with the flags of your choosing and save the output to an XML file. For example,
     ```
-    sudo nmap -sS -p80,443,8080,8443,1433,1521,27017 <ip-address> -oX my-network.xml
+    sudo nmap -sV -p80,443,8080,8443,1433,1521,27017 <ip-address/range> -oX my-network.xml
     ```
-    _Note: the **-sS** flag will enable TCP SYN port scan and the **-oX** specifies the output as a XML file. [Want more scanning options?](#nmap-usage)_
-    {% include tip.html content="Be sure to replace <ip-address> with the CIDR range for the network you would like to scan, ex. `10.0.0.0/24`" %}
+    [Go here to install Nmap](#installing-nmap).
+    
+    _Note: the **-sV** flag will attempt to determine the version of the service running on port and the **-oX** specifies the output as an XML file. [Want more scanning options?](#nmap-usage)_
 
+    {% include tip.html content="Be sure to replace <ip-address/range> with the CIDR range for the network you would like to scan, ex. `10.0.0.0/24`" %}
+ 
 4. Run this Tidal Tools command to upload your previously generated Nmap output to your Tidal Migrations account 
 ```
 tidal sync nmap <filename.XML> 
 ``` 
+5. Head over to your Tidal Migrations account! (https://<account_name>.tidalmg.com/#/labs/host_discovery)
  
-5. Head over to your Tidal Migrations account!
- 
-
-
 ## Installing Nmap
-
 Nmap ("Network Mapper") is a free and open source utility for network discovery and security auditing. Many systems and network administrators also find it useful for tasks such as network inventory, managing service upgrade schedules, and monitoring host or service uptime.
  
-You can find all the documentation in its official site [Nmap](https://nmap.org/) and instructions for how to download based on your environment [download](https://nmap.org/download.html)
+You can find all the documentation on [Nmap's](https://nmap.org/) official site  and instructions for how to download based on your environment [download](https://nmap.org/download.html).
```

Comment:
IMO, “**You can find all the documentation and instructions on how to download Nmap to your environment on the official site.**” looks better.

Reply:
yes, It does read better 👍